### DISCUSSION TITLE
Changing file paths to `/` instead of `\\` on Windows

### DISCUSSION THREAD

**Discussion by Jarartur** [retorquere/zotero-better-bibtex#2515] [Q&A] [answered]
Hi,
I wanted to ask if there is somewhere an option to export file paths in a more linux-compatible way. Currently, my paths are made up of `\\` as per Windows file paths. I would like to convert them to normal `/` path inside the `.bib` file.

Is there a way to enable this directly in zotero or better bibtex?

I figured I can always do in the postscript something like I found in the documentation, but would prefer not to if it is possible.

```
if (Translator.BetterTeX && !Translator.options.exportFileData && zotero.attachments && zotero.attachments.length) {
  for (const att of zotero.attachments) {
    if (att.localPath) {
      att.localPath = att.localPath.replace('\\', '/')
    }
  }
  tex.add({ name: 'file', value: zotero.attachments, enc: 'attachments' })
  return { cache: false }
}
```

**Comment by retorquere** [comment, marked as answer]
You'd need `att.localPath = att.localPath.replace(/\\/g, '/')`, but no, in BBT a postscript is the only way to do this. For standard Zotero you could in principle edit the bibtex translator.

**Reply by Jarartur** [reply]
Thanks very much :), I will do the postscript thing then.
